keyboards/handwired/brain/config.h

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+#define PRODUCT     Moduform
+
+#define SPLIT_USB_DETECT
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 7
+
+// wiring of each half
+
+#define MATRIX_COL_PINS { D1, D4, C6, D7, E6, B4, B5 }
+#define MATRIX_ROW_PINS { F4, F5, F6, F7, B1 }
+
+#define MATRIX_COL_PINS_RIGHT { B5, B4, E6, D7, C6, D4, D1 }
+
+#define DIODE_DIRECTION COL2ROW
+
+/* USB Device descriptor parameter */
```

Comment:
For readability, would prefer this go above, with the `PRODUCT` define.

Reply:
Is the the place I moved it better?